cmd/skopeo/copy.go

```diff
@@ -63,8 +64,15 @@ func copyHandler(context *cli.Context) error {
 }
 
 var copyCmd = cli.Command{
-	Name:      "copy",
-	Usage:     "Copy an image from one location to another",
+	Name: "copy",
+	Usage: fmt.Sprintf(`
```

Comment:
Actually, `Usage` seems to be expected to be an one-line string. This change results in:
```sh
$ ./skopeo help
<snip>
COMMANDS:
     copy	
		Copy an image from one transport location to another

	Supported transports:
	docker, docker-archive, docker-daemon, oci, atomic, ostree, containers-storage, dir

		See skopeo(1) section “IMAGE NAMES” for the expected format
		
     inspect	
		Inspect an image at a transport

	Supported transports:
	docker-archive, docker-daemon, oci, atomic, ostree, containers-storage, dir, docker

		See skopeo(1) section “IMAGE NAMES” for the expected format
<snip>
$ ./skopeo copy --help
NAME:
   skopeo copy - 
	Copy an image from one transport location to another

	Supported transports:
	containers-storage, dir, docker, docker-archive, docker-daemon, oci, atomic, ostree

	See skopeo(1) section “IMAGE NAMES” for the expected format
	

USAGE:
   skopeo copy [command options] SOURCE-IMAGE DESTINATION-IMAGE
<snip>
```
(instead of a single list of commands a weirdly-formatted list long descriptions, and command-specific help is all in the “Name” section)

It seems that `urfave/cli` wants to keep `Usage` one-line, and use `Description` for a long per-command help:
```sh
$ ./skopeo help
<snip>
COMMANDS:
     copy	Copy an image from one location to another
     inspect	Inspect image IMAGE-NAME
<snip>
$ ./skopeo copy --help
NAME:
   skopeo copy - Copy an image from one location to another

USAGE:
   skopeo copy [command options] SOURCE-IMAGE DESTINATION-IMAGE

DESCRIPTION:
   
	Copy an image from one transport location to another

	Supported transports:
	containers-storage, dir, docker, docker-archive, docker-daemon, oci, atomic, ostree

	See skopeo(1) section “IMAGE NAMES” for the expected format
	

OPTIONS:
<snip>
```